http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeServiceInterp.scala

```diff
@@ -232,34 +232,23 @@ class RuntimeServiceInterp[F[_]: Parallel](
 
   override def getRuntime(userInfo: UserInfo, cloudContext: CloudContext, runtimeName: RuntimeName)(implicit
     as: Ask[F, AppContext]
-  ): F[GetRuntimeResponse] =
-    for {
-      ctx <- as.ask
-      // throw 403 if no project-level permission
-      hasProjectPermission <- authProvider.isUserProjectReader(
-        cloudContext,
-        userInfo
-      )
-      _ <- F.raiseWhen(!hasProjectPermission)(ForbiddenError(userInfo.userEmail, Some(ctx.traceId)))
+  ): F[GetRuntimeResponse] = for {
+    ctx <- as.ask
 
-      // throws 404 if not existent
-      resp <- RuntimeServiceDbQueries.getRuntime(cloudContext, runtimeName).transaction
+    runtime <- RuntimeServiceDbQueries.getRuntime(cloudContext, runtimeName).transaction
 
-      // throw 404 if no GetClusterStatus permission
-      hasPermission <- authProvider.hasPermissionWithProjectFallback[RuntimeSamResourceId, RuntimeAction](
-        resp.samResource,
-        RuntimeAction.GetRuntimeStatus,
-        ProjectAction.GetRuntimeStatus,
-        userInfo,
-        GoogleProject(cloudContext.asString)
+    bearerToken = userInfo.accessToken.token
+
+    _ <- samService
+      .checkAuthorized(
+        bearerToken,
+        runtime.samResource,
+        RuntimeAction.GetRuntimeStatus
       )
-      _ <-
-        if (hasPermission) F.unit
-        else
-          F.raiseError[Unit](
-            RuntimeNotFoundException(cloudContext, runtimeName, "permission denied")
-          )
-    } yield resp
+      .adaptError { case _: SamException =>
```

Comment:
Do we need to adapt the error? `SamServiceInterp` already [throws](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/sam/SamServiceInterp.scala#L211) a friendly LeoException with status 403 -- can we just throw that?

Reply:
My understanding is that the best practice is to mask 403/forbidden style errors with 404 messages, since users without access rights shouldn't be able to verify that a runtime with the given ID even exists. This is how getRuntime worked before, and I think it's standard across Leo (and a pretty common pattern elsewhere)

Reply:
I feel like it's reasonable to respond 404 if the runtime doesn't exist and 403 if the runtime exists but the caller doesn't have permission.

Reply:
I'll ask AppSec what their preferred behavior is.

For context I think you wrote [this comment](https://github.com/DataBiosphere/leonardo/blob/b09387419d978b0ec783b9e2911cacf1c69a7f70/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeServiceInterp.scala#L327) ~4ya

Reply:
I think our comments crossed.. yeah, just thinking more about the best practice.

For instance, I was trying google apis and they seem to respond 403 in both cases:

```
$ gcloud projects describe asdfasdffoobar --verbosity debug

 User [rtitle@quality.firecloud.org] does not have permission to access projects instance [asdfasdffoobar] (or it may not exist):
…
  "error": {
    "code": 403,
    "message": "The caller does not have permission",
    "status": "PERMISSION_DENIED"
  }
```